core-api/src/main/resources/definitions/exposed-services.json

```diff
@@ -322,6 +322,20 @@
       "visibleForDatalake": true,
       "visibleForDatahub": true
     },
+    {
+      "name": "NIFI_REGISTRY_REST",
+      "displayName": "NiFi Registry Rest",
+      "serviceName": "NIFI_REGISTRY_SERVER",
+      "knoxService": "NIFI_REGISTRY_REST",
```

Comment:
This should be `NIFI-REGISTRY` and not `NIFI_REGISTRY_REST`. If you want this to change, most likely need to change here as well:

https://github.com/hortonworks/cloudbreak/blob/master/orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology_api.xml.j2#L202